lib/hashie/extensions/dash/coercion.rb

```diff
@@ -2,12 +2,14 @@ module Hashie
   module Extensions
     module Dash
       module Coercion
-        # Extends a Dash with the ability to define coercion for properties.
-
-        def self.included(base)
-          base.send :include, Hashie::Extensions::Coercion
+        includer = ::Hashie::Extensions::Coercion::CoercionSystemIncludeBuilder
```

Comment:
I thought we could still use `Hashie::Extensions::Coercion` like we used to for backward compatibility?

I think I'd rather keep the backward compatibility in Dash. What do you think @dblock?